views/message.erb

```diff
@@ -0,0 +1,4 @@
+<section class="message message--<%= kind %>">
```

Comment:
@niquerio I wasn't sure how to write this where the `message--[kind]` only applies when `kind` has a value 🤔

Reply:
So the `kind` prop for the `message` template is an option. Right now I just pass an empty string to keep it happy.

Reply:
Is the idea that `message.erb` would be used for more than just flash messages? Because if it's just for flash there will always be a `kind`

Reply:
Yeah, I think it could be used elsewhere. Good to know flashes will always have a kind.